### DISCUSSION TITLE
急需帮助！！！

### DISCUSSION THREAD

**Discussion by witherrose7564** [share121/inter-knot#5019] [委托]
oh god!我最近在网上购买了网传是法厄同用过的东西，但是那送货员又双叒叕把货送到了空洞......
希望有人可以帮我把货从空洞带出来吗？！！！
价格好说哦

**Comment by swyh01000** [comment]
这个委托我接了

**Reply by witherrose7564** [reply]
oh, 我马上把坐标发给你awa

**Comment by fzdqz** [comment]
@绳网团队 脚本接委托现象能不能管管啊

**Comment by zzz20100624** [comment]
这个任务我们治安局接了🤓

**Comment by IXfromThorne** [comment]
诈骗手法千万条，提高警惕第一条。
购买渠道不规范，徒留钱包两行泪。
——被迫营业的空洞特勤组伊克斯 宣

**Comment by KELERSAMA** [comment]
都说了天马速运不行 包裹会丢

**Comment by cuijiaqi0406** [comment]
服了这个™速运，上次把我的以骸运到空洞里了

**Reply by witherrose7564** [reply]
啊？！这玩意不是原本就在空洞里吗？(
